Formula/metricbeat.rb

```diff
@@ -74,6 +74,11 @@ def install
     end
 
     sleep 30
-    assert_predicate testpath/"data/metricbeat", :exist?
+    assert_match "first_start", File.read(testpath/"data/meta.json")
+
+    # test against `metricbeat-<calendar>.ndjson`
+    Dir.glob(testpath/"data/metricbeat-*.ndjson").each do |file|
+      assert_match "\"beat\":\"metricbeat\"", File.read(file)
+    end
```

Comment:
Can we do `JSON.parse` to make sure that this is valid JSON? Also, use
```ruby
(testpath/"data/meta.json").read
```
instead of `File.read` and
```ruby
(testpath/"data/metricbeat-*.ndjson").glob
```
instead of `Dir.glob`.

Reply:
Sounds good

Reply:
Thanks for the suggestions, it should be in better shape now. 👍